tests/lamp_stack/pgbench/README.md

```diff
@@ -20,3 +20,9 @@ To run pgbench only
 # on consecutive runs
 ./run_pgbench_2.sh
 ```
+
+To plot the charts have a look at all the `gen_X_plots.py` scripts. To know how to run all these plotting scripts have a look at `plot.sh`
+```sh
+./plot.sh
```

Comment:
I think this and the lamp/nginx readmes need a bit more explanation here. Mainly that you have to create a data directory and what the format for the json is. Maybe actually add the directory with sample jsons in them being like pgbench-data-example.json etc

Reply:
Yeah that makes sense. I have updated the README.md and also added a few sample JSONs, could you please review it again?